client/js/libs/jquery/stickyscroll.js

```diff
@@ -1,11 +1,19 @@
 (function($) {
 	$.fn.unsticky = function() {
+		$(window).off("resize.sticky", this.data("window-sticky-fn"));
 		return this.unbind(".sticky");
 	};
 
 	$.fn.sticky = function() {
 		var self = this;
 		var stuckToBottom = true;
+		var windowResizeHandler = function() {
```

Comment:
Rename it to something more generic and make `"msg.sticky"` event use this callback too (it's same code).